src/collective/nitf/tests/test_setup.py

```diff
@@ -18,16 +18,24 @@
 CSS = '++resource++collective.nitf/nitf.css'
 
 
-class InstallTestCase(unittest.TestCase):
+class InstallTestCase(unittest.TestCase, QIBBB):
```

Comment:
how about to make class ``QIBBB`` inherit from ``unittest.TestCase``, and all other tests inherit from ``QIBBB``?